Modules/_testcapi/parts.h

```diff
@@ -31,6 +31,8 @@
 int _PyTestCapi_Init_Vectorcall(PyObject *module);
 int _PyTestCapi_Init_Heaptype(PyObject *module);
 int _PyTestCapi_Init_Abstract(PyObject *module);
+int _PyTestCapi_Init_ByteArray(PyObject *);
+int _PyTestCapi_Init_Bytes(PyObject *);
```

Comment:
```suggestion
int _PyTestCapi_Init_ByteArray(PyObject *module);
int _PyTestCapi_Init_Bytes(PyObject *module);
```

Just for consistency